state/modeluser.go

```diff
@@ -193,6 +193,19 @@ func (e *UserModel) LastConnection() (time.Time, error) {
 	return lastConnDoc.LastConnection, nil
 }
 
+// AllUserModels returns a list of all models with the user they belong to.
+func (st *State) AllUserModels() ([]*UserModel, error) {
+	models, err := st.AllModels()
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	var result []*UserModel
+	for _, model := range models {
+		result = append(result, &UserModel{Model: model, User: model.Owner()})
```

Comment:
If you can just call Model.Owner(), why do we need a separate UserModel type? Isn't it just exactly what you have here, which is model.Owner()?